unittests/DefToYAMLConverter/DefToYAMLConverterTest.cpp

```diff
@@ -38,6 +45,9 @@ static std::string getDefaultLocalizationPath() {
   return std::string(DefaultDiagnosticMessagesDir.str());
 }
 
+/// Random number in [0,n)
+unsigned randNum(unsigned n) { return unsigned(rand()) % n; }
```

Comment:
I figured out that I can use this message since it's already has been used here https://github.com/apple/swift/blob/master/utils/test-clustered-bit-vector/generator.cpp#L12